gpkit/tests/t_geometric_program.py

```diff
@@ -91,9 +91,15 @@ def test_Mddtest(self):
         self.assertEqual(gp2.A, CootMatrix(row=[0, 1],
                                            col=[0, 0],
                                            data=[-1, 1]))
-        self.assertEqual(gp3.A, CootMatrix(row=[0, 2, 3],
-                                           col=[0, 0, 0],
-                                           data=[-1, 1, -1]))
+        # order of variables with a posynomial is not stable
+        #   (though monomial order is)
```

Comment:
This makes me wonder whether we need a semi-convenient way to get the A matrix corresponding to a certain variable (column) ordering, e.g. `gp.A(vars=('x', 'y', 'z'))`. Or is that silly?

Is the order of monomial terms (rows/constraints) stable?